lib/URLAssetsConstants.js

```diff
@@ -0,0 +1,31 @@
+// TODO:
+
+const URLAssetsConstants = {
+  base:'https://raw.githubusercontent.com/Statscell/brawl-assets/720b691d7ed3984b6724e83e4a541b4d355adbd5',
```

Comment:
```suggestion
  base:'https://raw.githubusercontent.com/Statscell/brawl-assets/main',
```
Can be simply done this so that there will be no need for committing a change to get latest assets

Reply:
I could. But I want to avoid a situation where the owner of this repository could swap the images and break the library.

Reply:
Thanks for reviewing my code btw!

Reply:
Yeah you can fork the repository and use the forked repository URL.